README.md

```diff
@@ -1,78 +1,66 @@
+* [Keboola documentation for developers](https://developers.keboola.com/overview/)
+* [Keboola API blueprint](https://kebooladocker.docs.apiary.io/#reference/actions/run-custom-component-action/process-action)
 
-Before run, you need to have one of configuration in `path_to_data_folder` folder.
+----------
 
-#### To run a crawler
+## Local Development
 
-```
-{
-  "parameters": {
-    "userId": "Apify user ID",
-    "#token": "Apify API token"
-    "crawlerId": "H8Xo8nhNDHJkmEfJG",
-    "crawlerSettings": {}
-  }
-}
-```
+If you are interested in adding a new feature or fixing a bug in the integration, feel free to open a pull request.
 
-#### To get results from existing execution
+Before you started developing this integration, you will need your Apify API token and user ID.
+You can find the token and ID [on the Integrations page of your Apify account](https://my.apify.com/account#/integrations).
 
-```
-{
-  "parameters": {
-    "executionId": "Execution Id"
-  }
-}
-```
+### Run Configuration
 
-#### To list crawlers:
-```
+The run of integration depends on configuration. You can specify which action you want to run and its parameters in the configuration file `config.json`.
+On the Keboola platform, will be this config generated for each run based on user input.
+The base configuration looks:
+```json
 {
-  "action": "listCrawlers",
+  "action": "run",
   "parameters": {
-    "userId": "Apify user ID",
-    "#token": "Apify API token"
+    "actionType": "runActor",
+    "userId": "myUserId",
+    "#token": "myToken",
+    "actId": "my-user-name/actor",
+    "input": {
+      "pages": 1
+    },
+    "memory": "512",
+    "build": "latest"
   }
 }
 ```
+With this configuration, you will run the actor with specific options.
+There is [folder](./test/configs) with all possible examples of configuration.
 
-### Run in Docker
-
-`docker build --tag=test-ex .`
-
-`docker run --volume=path_to_data_folder:/data/ test-ex`
-
-### Run without docker
 
-`npm run dev`
+### Run Action locally
 
-### Run docker image on top of Keboola platform
-Follow instruction on https://developers.keboola.com/extend/component/deployment/#test-live-configurations.
+1. Install dependencies `npm i`
+2. Create empty data dir for input and output of action `mkdir -p ./.temp/data`
+3. In data dir create `config.json` file and fill with an action you want to run `cp ./test/configs/actor_run.json ./.temp/data/config.json`
+4. Run the action using `npm run dev`
 
-### Run tests in node js app
+## Tests
 
-`DATA_DIR=path_to_data_folder APIFY_TEST_USER_ID=testUserIs APIFY_TEST_TOKEN=testUserToken npm run test`
+There are integrations tests, which check each action if it works with the Apify platform as expected.
+You can run these tests with your Apify token and user ID. You can find the token and ID [on the Integrations page of your Apify account](https://my.apify.com/account#/integrations).
```

Comment:
Maybe add where to find the user ID here as well pls

Reply:
It is on same page, I will update text